cassandra_test.go

```diff
@@ -2913,7 +2936,7 @@ func TestDiscoverViaProxy(t *testing.T) {
 	// This (complicated) test tests that when the driver is given an initial host
 	// that is infact a proxy it discovers the rest of the ring behind the proxy
 	// and does not store the proxies address as a host in its connection pool.
-	// See https://github.com/gocql/gocql/issues/481
+	// See https://github.com/apache/cassandra-gocql-driver/issues/481
```

Comment:
Do you plan to rename this github repository to `apache/cassandra-gocql-driver` (which should create a redirect to the new location) or create a new fork and archive this project? If fork, do you plan to transfer the list of issues?

I'd like to understand if new URL will work after the transfer.

Reply:
Once the donation is voted upon and accepted, we will then ask ASF Infra to move this github repository.  

The best approach, if possible, is to assign ASF Infra admin privileges to the repo and they will make the move (transfer organisation and rename project).  With this approach GitHub automatically handles redirects (both in browser and from git on the command line, and in tools).

Reply:
Thank you! That resolves my question.